contrib/bdii/htcondor-ce-provider

```diff
@@ -119,6 +125,11 @@ def main():
         sys.stderr.write("Error: GLUE2DomainID: not set\n")
         sys.exit(1)
 
+    # Get the timeout value
+    time_out = htcondor.param.get('CE_QUERY_TIMEOUT')
```

Comment:
Can we change this to `BDII_CE_QUERY_TIMEOUT` and add `BDII_CE_QUERY_TIMEOUT = 10` to each of the config files in the `bdii` dir?

Reply:
Could you expand on that? I was under the impression that GLUE was basically the schema and BDII is the system by which GLUE information is reported. So maybe BDII is a misnomer for this entire package but I'd rather have the naming be consistent for the user.

Reply:
The BDII is an information collection/aggregation service. GLUE is an information model and serialization into different data formats. This code is an information provider that queries the HTCondor batch system and returns information according to the GLUE2 information model in the LDIF.  I hope that is clear. I hope that is clear so yes the directory should be glue rather than bdii. For example you could imaging this provider being modified in the future to output the information in the JSON format to be compatible with CRIC.

Reply:
I would think that if we were to support the JSON/CRIC case, we'd call it the CRIC provider because that better describes the schema and the consumer of the information. If this provider starts doing double-duty, we should probably rename it all to `htcondor-ce-info-provider` or something along those lines.

Does anyone use GLUE2 outside of BDII?

Reply:
Yes, if the information model was specific to CRIC then yes it would be a different provider. GLUE2 is used outside of the BDII. There are official XML and JSON rendering documents.

Reply:
Sorry, I meant that we'd call it the "CRIC provider" instead of the "JSON provider". I think the same logic works here, especially since BDII is the only consumer of GLUE2 information: although the CE query is for constructing GLUE2-compliant data, it is ultimately for BDII service. I would think the end-user would be more concerned/aware of the service over the data format itself so it'd be friendlier to name the package, config, etc. after the service.